### PR TITLE
feat: Clear cache on exit

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

File: electron/src/runtime/lifecycle.ts [hunk 1]
```diff
@@ -69,15 +69,17 @@ export const getWebViewId = (contents: WebContents): string | undefined => {
 // Using exit instead of quit for the time being
 // see: https://github.com/electron/electron/issues/8862#issuecomment-294303518
 export const quit = (): void => {
-  logger.info('Initiating app quit ...');
+  logger.info('Initiating app quit...');
   settings.persistToFile();
+  logger.info('Clear cache...');
+  session.defaultSession.clearCache().catch(() => {});
```
Comment (lipis) [inline electron/src/runtime/lifecycle.ts]: that's the actual change!

Comment (bennyn) [reply electron/src/runtime/lifecycle.ts]: Ok, then let's keep this PR to that change and put back the spaces to "...". These were set on purpose.

Comment (bennyn) [inline electron/src/runtime/lifecycle.ts]:
```suggestion
  session.defaultSession.clearCache().catch(() => {});
```

There should be at least an error log when catching. Just swallowing the error is not a good practice.

Comment (lipis) [reply electron/src/runtime/lifecycle.ts]: From the copy point of view we should never have spaces before `,.?! etc`.

Shall we fix that in separate PR?

File: electron/src/runtime/lifecycle.ts [hunk 2]
```diff
@@ -71,6 +71,10 @@ export const getWebViewId = (contents: WebContents): string | undefined => {
 export const quit = (): void => {
   logger.info('Initiating app quit ...');
   settings.persistToFile();
+  logger.info('Clear cache ...');
+  session.defaultSession.clearCache().catch(error => {
+    logger.info(error);
```
Comment (bennyn) [inline electron/src/runtime/lifecycle.ts]: Let's flag it as an error or at least a warning.

File: electron/src/runtime/lifecycle.ts (hunk 1 above)
Comment (bennyn) [reply electron/src/runtime/lifecycle.ts]: No. As I wrote. The spaces were set on purpose. They should be kept.

File: electron/src/runtime/lifecycle.ts (hunk 2 above)
Comment (lipis) [reply electron/src/runtime/lifecycle.ts]: done..

**Pull request merged**